schema.development.kf

```diff
@@ -448,16 +449,27 @@ action get_credential_owned ($id) public view {
 @kgw(authn='true')
 procedure get_credential_shared ($id uuid) public view returns table (id uuid, human_id uuid, credential_type text,
     credential_level text, credential_status text, content text, encryption_public_key text, issuer text, inserter text) {
-    if !shared_credential_exist($id) {
-        error('shared credential does not exist');
+    if !credential_exist($id) {
+        error('the credential does not exist');
     }
 
-    if !data_granted_to_grantee($id, @caller) {
-        error('shared credential is not shared with the caller');
+    $granted bool := false;
+    $grant_valid bool := false;
+
+    for $row in SELECT status FROM access_grants WHERE data_id = $id::TEXT AND ag_grantee = @caller COLLATE NOCASE {
+        $granted := true;
+        if $row.status == 'valid' {
```

Comment:
I feel like we're missing an else here. If not, where do we set the `status` to `true`?

Reply:
`status` is [set in the oracle](https://github.com/idos-network/idos-kgw/pull/16/files#diff-812123e4e094d4bff4f76c8f92a90a0647a931a455a70f38bbf9b69a67a0faf5R105-R129)
In `get_credential_shared` we only use its value to check.